fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -177,6 +177,8 @@ class FMCMainDisplay extends BaseAirliners {
         this.arincLandingElevation = FmArinc429OutputWord.empty("LANDING_ELEVATION");
         this.arincDestinationLatitude = FmArinc429OutputWord.empty("DEST_LAT");
         this.arincDestinationLongitude = FmArinc429OutputWord.empty("DEST_LONG");
+        this.arincMDA = FmArinc429OutputWord.empty("MDA");
+        this.arincDH = FmArinc429OutputWord.empty("DH");
```

Comment:
Please add these to the `a320-simvars.md` doc.

I did something very similar to this in the `rust-fwc` branch, but I'm happy to follow your convention after this merged:
https://github.com/flybywiresim/a32nx/blob/0929395f8424ff5668576245f72806c23cf2b014/fbw-a32nx/docs/a320-simvars.md?plain=1#L1212-L1223